doit/runner.py

```diff
@@ -543,13 +539,12 @@ def execute_task_subprocess(self, job_q, result_q, reporter_class):
 
                 result = {'name': task.name}
                 t_result = self.execute_task(task)
-
-                if t_result is None:
-                    result['task'] = task.pickle_safe_dict()
-                    result['out'] = [a.out for a in task.actions]
-                    result['err'] = [a.err for a in task.actions]
-                else:
+                if t_result:
```

Comment:
`t_result` is actually a misleading variable name. It only has a value if there was a failure.
What about changing it to `t_failure` ?

Reply:
Totally agree, I'll change it.

Reply:
Done